### PR TITLE
Switch component renamed to Choice

### PR DESCRIPTION
**Pull request by arkadius**

### CODE REVIEW

File: ui/client/assets/json/nodeAttributes.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     }
   },
   "Switch": {
```
Comment (lciolecki) [inline ui/client/assets/json/nodeAttributes.json:37]: What about this?

Comment (arkadius) [reply ui/client/assets/json/nodeAttributes.json:37]: Switch component type (in scenario json) stay with us. We change only displayed labels on FE.

Comment (lciolecki) [reply ui/client/assets/json/nodeAttributes.json:37]: Ehh this will be misleading..

File: docs/scenarios_authoring/BasicNodes.md [hunk 2]
```diff
@@ -10,6 +10,32 @@ Most of the nodes, with source and sink nodes being notable exceptions, have inp
 
 Sinks and filters can be disabled by selecting `Disable` checkbox. 
 
+## Choice
```
Comment (dswiecki) [inline docs/scenarios_authoring/BasicNodes.md:13]: I'll put it after variable and mapVariable, just as it was previously

**Review by dswiecki — APPROVED** [review]
LGTM

**Pull request merged**